docker-compose.prod.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+services:
+  web:
+    container_name: tracemap-api
+    build: .
+    ports:
+     - '5000:5000'
+     - '5100:5100'
+    env_file: .env
```

Comment:
I've created a separate file for prod config, so we leave it easy for developers to develop with a simple `docker-compose up`, while still being able to deploy with `docker-compose up -f docker-compose.prod.yml`